it/controllers/InitialiseAppealControllerISpec.scala

```diff
@@ -173,4 +174,27 @@ class InitialiseAppealControllerISpec extends IntegrationSpecCommonBase {
       await(result).header.status shouldBe INTERNAL_SERVER_ERROR
     }
   }
+
+  "GET /initialise-appeal-against-the-obligation-estimated-lpp" should {
+    "redirect to Cancel VAt Page" in new Setup {
+      implicit val fakeRequest: FakeRequest[AnyContentAsEmpty.type] = FakeRequest().withSession(
+        authToken -> "1234"
+      )
+      val result: Future[Result] = controller.onPageLoadForObligationEstimatedLPP("2025-01-01", "2025-03-31")(fakeRequest)
+      await(result).header.status shouldBe SEE_OTHER
+      redirectLocation(result).get shouldBe routes.CancelVATRegistrationController.onPageLoadForCancelVATRegistration().url
+      val userAnswers = await(userAnswersRepository.collection.find(Document()).toFuture()).head
+      userAnswers.getAnswer[LocalDate](SessionKeys.startDateOfPeriod).isDefined shouldBe true
+      userAnswers.getAnswer[LocalDate](SessionKeys.endDateOfPeriod).isDefined shouldBe true
```

Comment:
We should have dueDate, penaltyNumber, journeyId and commsDate in the session, could there be a test for that?

Reply:
Added ✅